### PR TITLE
#599 and #601 done

### PR DESCRIPTION
**Pull request by shreyanema33**
Added new pages and scrollBar added to the editor.

### CODE REVIEW

File: src/pages/Editor/containers/editContext.js [hunk 1]
```diff
@@ -31,18 +31,19 @@ const EditContextProvider = props => {
...
-    Ruled2: "ruled2.jpg",
-    OnlyMargin: "onlymargin.jpg",
-    Blank1: "blank1.png",
-    Blank2: "blank2.jpg",
+    Ruled1: "ruled1.jpg",
+    Ruled2: "ruled2.jpeg",
+    OnlyMargin: "margin1.png",
+    Blank1: "blank1.jpg",
+    Diary1: "diary1.jpg",
+    Diary2: "diary2.jpg"
```
Comment (smaranjitghose) [inline src/pages/Editor/containers/editContext.js:39]: Only add the new files, Do not tamper with the existing ones

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by anushbhatia** [comment]
Add the scroll bar on the home page and the editor page, not inside the editor that looks weird and not really helpful.

**Review by anushbhatia — CHANGES_REQUESTED** [review]
Resolve the conflict and also add blue colour to the scrollbar so that it resembles our theme and keep the scrollbar uniform on all pages

**Comment by smaranjitghose** [comment]
## Feedback:
- Diary2 is similar to Ruled4. We don't need that
- Blank2 is irrelevant for this scenario.

Please do a squash commit and raise a new PR following the contribution guidelines carefully

**Pull request closed without merging**